content/docs/integrations/gcalendar/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Google Calendar Plugin 
```

Comment:
Should this be "Home Page Google Calendar" to match "Home Page Markdown"?

I'm just thinking it's a good idea to have these home page widgets collected together in the sidebar.

Reply:
While currently it only supplies a home page card the review when it was introduced  asked that it be named generically so any future calendar widgets could be added here (I think even gsuite was suggested). We probably should have done that with the home page markdown plugin too.

Reply:
This doesn't stop us linking to the "add google calendar to the home page" from a getting started page around the home page for example or from next steps in the current home page docs